apps/breethe_web/lib/breethe_web/endpoint.ex

```diff
@@ -42,6 +42,7 @@ defmodule BreetheWeb.Endpoint do
     signing_salt: "KUZ/RHEu"
   )
 
+  plug(CORSPlug)
```

Comment:
@niklaslong: I guess removing this might have been wrong?

Reply:
🤔 not sure why though...

Reply:
would the plug be added automatically somehow when the config is present?

Reply:
Ah yes, checked the docs again - I should have left the plug in the endpoint.